docs/modules/deployment/pages/upgrade/backup/introduction.adoc

```diff
@@ -0,0 +1,35 @@
+[[backup]]
+= Backup OpenNMS {page-component-title}
+
+OpenNMS {page-component-title} configurations and performance data storage methods can differ a lot depending on the customer needs.
+This article describes the backup process of a default installation of OpenNMS {page-component-title}.
```

Comment:
```suggestion
This section describes the backup process of a default installation of OpenNMS {page-component-title}.
```

Reply:
This is a good place to define what a default installation looks like. (One instance, no Minions, Sentinel, etc.) Do we add a link to the directory structure as described in core instance install? (https://docs.opennms.com/horizon/29/deployment/core/getting-started.html#install-core-instance)